.github/ISSUE_TEMPLATE/01-sdk-bug.yml

```diff
@@ -0,0 +1,100 @@
+name: SDK Bug Report
+description: File SDK bug report
+labels: ["type:bug", "status:waiting-for-triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Thank you for taking the time to fill out this bug report!**
+        💥Before submitting a new request, please search existing issues to see if an issue already exists.
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: |
+        Provide a description of the actual behavior observed. If applicable please include any error messages, exception stacktraces or a screenshot.
```

Comment:
Should error messages and stack traces be in the debug section below?

Reply:
That is optional. If someone feels that the easiest way to describe the bug is through stacktrace or error message or screenshot, then that should be allowed too. They can skip the debug section below if this happens